shell/common/shell.h

```diff
@@ -377,7 +379,8 @@ class Shell final : public PlatformView::Delegate,
   fml::WeakPtr<Engine> weak_engine_;          // to be shared across threads
   fml::WeakPtr<Rasterizer> weak_rasterizer_;  // to be shared across threads
   fml::WeakPtr<PlatformView>
-      weak_platform_view_;  // to be shared across threads
+      weak_platform_view_;           // to be shared across threads
+  fml::TimeRecorder time_recorder_;  // to be shared across threads
```

Comment:
Nit: Careful with the destruction order. I'd move this above the ivars for any component that may end up using this (or just give them their own shared pointer to this object). So at least before the unique pointers to the primary thread specific components. In this specific case, it does not matter because we reset the components in the destructor but its just good to follow a safe pattern.